lua/wikis/wildcard/Infobox/Unit/Custom.lua

```diff
@@ -0,0 +1,61 @@
+---
+-- @Liquipedia
+-- wiki=wildcard
+-- page=Module:Infobox/Unit/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+
+local Injector = Lua.import('Module:Widget/Injector')
+local Unit = Lua.import('Module:Infobox/Unit')
+
+local Widgets = require('Module:Widget/All')
+local Cell = Widgets.Cell
+
+---@class WildcardUnitInfobox: UnitInfobox
+local CustomUnit = Class.new(Unit)
+local CustomInjector = Class.new(Injector)
+
+---@param frame Frame
+---@return Html
+function CustomUnit.run(frame)
+	local unit = CustomUnit(frame)
+	unit:setWidgetInjector(CustomInjector(unit))
+	unit.args.informationType = 'Summon'
+	return unit:createInfobox()
+end
+
+---@param id string
+---@param widgets Widget[]
+---@return Widget[]
+function CustomInjector:parse(id, widgets)
+	local args = self.caller.args
+	if id == 'custom' then
+		return Array.append(widgets,
+			Cell{name = 'Pronouns', content = {args.pronouns}},
+			Cell{name = 'House', content = {args.house}},
+			Cell{name = 'Rarity', content = {args.rarity}},
+			Cell{name = 'Mana Cost', content = {args.cost}},
+			Cell{name = 'Quantity', content = {args.quantity}},
+			Cell{name = 'Health', content = {args.health}},
+			Cell{name = 'Movement Speed', content = {args.movement}},
+			Cell{name = 'Key Words', content = {args.keywords}}
+		)
+	end
+	return widgets
+end
+
+---@param args table
+function CustomUnit:setLpdbData(args)
+	mw.ext.LiquipediaDB.lpdb_datapoint('unit_' .. self.pagename, {
+		name = args.name,
+		type = args.informationType,
+		extradata = mw.ext.LiquipediaDB.lpdb_create_json{house = args.house or '',},
+	})
+end
```

Comment:
```suggestion
---@param lpdbData table
---@param args table
---@return table
function Character:addToLpdb(lpdbData, args)
	lpdbData.extradata.house = args.house or '',

	return lpdbData
end
```

Reply:
Unit doesn't have default lpdb

Reply:
it seems to not work here, Since this is **Unit** infobox but I need to invoke the **Character** base module too? (I've tried this revision but ends up losing all stored LPDB data https://liquipedia.net/wildcard/index.php?title=Module%3AInfobox%2FUnit%2FCustom&type=revision&diff=840&oldid=778, `house` from args.house and `type` from the informationType are the two I need)

Reply:
ah my bad, thought it was character